website/templates/emails/digest_reviews_moderators.html.mako

```diff
@@ -5,13 +5,29 @@
 <%def name="build_message(d)">
     <table class="block" width="100%" border="0" cellpadding="15" cellspacing="0" align="center">
         <thead class="block-head">
-            <tr>
-                <th colspan="2" style="padding: 0px 15px 0 15px">
-                    <h3 style="padding: 0 15px 5px 15px; margin: 30px 0 0 0;border: none;list-style: none;font-weight: 300; border-bottom: 1px solid #eee; text-align: left;">
-                        Visit your <a href=${reviews_submissions_url}>provider’s submissions</a>
-                    </h3>
-                </th>
-            </tr>
+            % if provider_type == 'preprint':
```

Comment:
Is provider_type added to the context somewhere? I'm not seeing it

Reply:
See website/templates/emails/digest_reviews_moderators.html.mako file changes